lib/mongoid/interceptable.rb

```diff
@@ -162,6 +170,46 @@ def _mongoid_run_child_callbacks(kind, children: nil, &block)
       end
     end
 
+    def _mondoid_run_child_before_callbacks(kind, children: [], callback_list: [])
```

Comment:
mondoid --> mongoid

Reply:
Fixed, thank you. Mondoid sounds kind of nice, though...

Reply:
Mondoid does sound kind of nice